### DISCUSSION TITLE
How can we get real time visual debugger?

### DISCUSSION THREAD

**Discussion by MunoDevelop** [NVIDIA-Omniverse/PhysX#261] [Q&A]
i basically writing drawing part, omni pvd not work for me, it looks like pvd to file and read file after simulate(and read part not work) , does physX support simple real time visual debug like "just draw all existing actors runtime"? so no need to write everything about openGl

**Comment by preist-nvidia** [comment]
You can use the debug visualization, maybe? https://nvidia-omniverse.github.io/PhysX/physx/5.3.1/docs/DebugVisualization.html

Or do you need the custom drawing functionality of the deprecated PVD?
